cognite/neat/rules/exporters/_rules2excel.py

```diff
@@ -234,82 +238,67 @@ def _adjust_column_widths(cls, workbook: Workbook) -> None:
-        if isinstance(existing_metadata["updated"], datetime):
-            metadata["updated"] = existing_metadata["updated"].replace(tzinfo=None)
-        # Excel does not support timezone in datetime strings
-        now_iso = datetime.now().replace(tzinfo=None).isoformat()
-        is_info = isinstance(rules, InformationRules)
-        is_dms = isinstance(rules, DMSRules)
-        is_extension = self.new_model_id is not None or rules.reference is not None
-        is_solution = rules.metadata.data_model_type == DataModelType.solution
-
-        if is_solution and self.new_model_id:
-            metadata["prefix" if is_info else "space"] = self.new_model_id[0]  # type: ignore[index]
-            metadata["title" if is_info else "externalId"] = self.new_model_id[1]  # type: ignore[index]
-            metadata["version"] = self.new_model_id[2]  # type: ignore[index]
-        elif is_solution and self.dump_as == "reference" and rules.reference:
-            metadata["prefix" if is_info else "space"] = "YOUR_PREFIX"
-            metadata["title" if is_info else "externalId"] = "YOUR_TITLE"
-            metadata["version"] = "1"
-        else:
-            metadata["prefix" if is_info else "space"] = existing_model_id[0]
-            metadata["title" if is_info else "externalId"] = existing_model_id[1]
-            metadata["version"] = existing_model_id[2]
-
-        if is_solution and is_info and self.new_model_id:
-            metadata["namespace"] = f"http://purl.org/{self.new_model_id[0]}/"  # type: ignore[index]
-        elif is_info:
-            metadata["namespace"] = existing_metadata["namespace"]
-
-        if is_solution and is_dms and self.new_model_id:
-            metadata["name"] = self.new_model_id[1]  # type: ignore[index]
-
-        if is_solution:
-            metadata["created"] = now_iso
-        else:
-            metadata["created"] = existing_metadata["created"]
-
-        if is_solution or is_extension:
-            metadata["updated"] = now_iso
-        else:
-            metadata["updated"] = existing_metadata["updated"]
 
-        if is_solution:
-            metadata["creator"] = "YOUR NAME"
-        else:
-            metadata["creator"] = existing_metadata["creator"]
-
-        if not is_solution:
-            metadata["description"] = existing_metadata["description"]
+class _MetadataCreator:
+    creator_name = "<YOUR NAME>"
 
-        if is_extension:
-            metadata["schema"] = SchemaCompleteness.extended.value
-        else:
-            metadata["schema"] = SchemaCompleteness.complete.value
-
-        if is_solution:
-            metadata["dataModelType"] = DataModelType.solution.value
+    def __init__(
+        self,
+        action: Literal["create", "update"],
+        new_model_id: tuple[str, str] | None = None,
+    ):
+        self.action = action
+        self.new_model_id = new_model_id or ("YOUR_PREFIX", "YOUR_TITLE")
```

Comment:
so in case of InformationRules prefix goes to space, title as an external id. 
My worry is that title can have characters not compliant with external_id of model.

Reply:
What makes new id for information model is actually `namespace`, as it is a unique id of data model.
So in case of Information model new id would be "YOUR_PREFIX" "YOUR_NAMESPACE"

Reply:
As it is set up now, maybe we are fine, as you are using `prefix` to create `namespace`